### ISSUE TITLE
Click on Reaunthenticate button - nothing happen

### ISSUE THREAD

**Issue by mkz212** [hassio-addons/addon-tailscale#442]
# Problem/Motivation

Click on Reaunthenticate button - nothing happen. Using iPhone. I need to deactivate option in Safari - Block pop-up windows.

## Expected behavior

Best if this will be not popup window. But.. At least add note under Reautenthicate button to warn that popup windows must be not blocked. 

## Actual behavior

> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by lmagyar** [comment]
As the docs says: **Note:** _Some browsers don't work with this step. It is recommended to complete this step on a desktop or laptop computer using the Chrome browser._

**Comment by jasonhollis** [comment]
The issue is not stale.  It's still broken.  I push the reauthenticate button and nothing happens in Safari or Chrome.

**Comment by mkz212** [comment]
> The issue is not stale. It's still broken. I push the reauthenticate button and nothing happens in Safari or Chrome.

You need to allow pop-up windows. You can enable this in Safari settings. There should be info under the Reauthenticate button.

**Comment by jasonhollis** [comment]
Nope, I remember that from the past.  I have no option to disable the pop-ups in either browser because I'm not getting that far.

**Comment by jasonhollis** [comment]
https://github.com/user-attachments/assets/fa4ccd57-10c4-4120-9ad4-e4c384901ee5

**Comment by mkz212** [comment]
> Nope, I remember that from the past. I have no option to disable the pop-ups in either browser because I'm not getting that far.

In Safari settings, you need to allow pop-ups. And only then click this reauthenticate button.

**Comment by jasonhollis** [comment]
They are disabled:

![Image](https://github.com/user-attachments/assets/481363b1-4f13-44b6-b7da-8c2fcff45c56)

No change.

**Comment by jasonhollis** [comment]
https://github.com/user-attachments/assets/222055a8-c485-4978-8ca0-9fce24179ec4

**Comment by lmagyar** [comment]
Enable popups everywhere (for other websites), authenticate, disable popups where you want.
And disable key expiry for this add-on for the future.

**Comment by jasonhollis** [comment]
I have disabled key expiry already.  I have disabled popups across the board and the button to reauthenticate still does nothing.  I'm familiar with the tailscale CLI.  I just don't know how to navigate to it through the container substructures in HAOS.  If you can tel me how to get to it in the terminal I'm sure I can figure out how to get it hooked back up that way.

**Comment by lmagyar** [comment]
Enable popups!

You can also find the auth url in the add-on logs.

**Comment by jasonhollis** [comment]
Mate,

You are not listening.

![Image](https://github.com/user-attachments/assets/b651cb13-dfb1-4b94-b3fb-3addc9b9f9d9)

The only way I could make this work is to hit the auth URL from my Mac.  I'm never prompted about the popups when it's set that way and it does not run with the popup blocker disabled.
